persistence/trees/module.go

```diff
@@ -8,15 +8,17 @@ import (
 	"github.com/pokt-network/smt"
 )
 
+var _ modules.TreeStoreModule = &treeStore{}
+
 func (*treeStore) Create(bus modules.Bus, options ...modules.TreeStoreOption) (modules.TreeStoreModule, error) {
 	m := &treeStore{}
 
+	bus.RegisterModule(m)
```

Comment:
I did a search for `for _, option := range options {` in the codebase and found that we always do `RegisterModule` AFTER applying the options.

I know it's unlikely that the options will impact registration (maybe with the exception of tests/mocks) but just want to stay consistent.

Reply:
Noted! I will add this to the module documentation 👍